.github/workflows/dotnet-core.yml

```diff
@@ -20,11 +20,16 @@ jobs:
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: 5.0.101
-    - name: Install dependencies
-      run: dotnet restore
+    - name: Set up CodeClimate Test Reporter
+      run: |
+        curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./codeclimate-test-reporter
+        chmod +x ./codeclimate-test-reporter
+        ./codeclimate-test-reporter before-build
```

Comment:
There is not an official GitHub Action for Code Climate but there is one recommended by Code Climate:

https://github.com/paambaati/codeclimate-action

I'm usually hesitant to use non-official Actions but given that this is recommended it could be worth checking out.  It also appears to fix an issue with Code Climate and GitHub Actions Test Coverage:

https://docs.codeclimate.com/docs/github-actions-test-coverage